### ISSUE TITLE
Add more hooks

### ISSUE THREAD

**Issue by ciembor** [git-hooks/git-hooks#24]
I was trying to use git hooks to take some actions on a database during changing a branch (`checkout`), creating a branch (`branch`), deleting branch (`branch -d`) etc. Is there a specific reason why such hooks are not available? It would be very handy to have them!

**Comment by rumenvasilev** [comment]
Because they simply don't exist. Here's a complete list of hooks: https://githooks.com

**Comment by ciembor** [comment]
@rumenvasilev I know they don't exist. So this issue is about implementing them :). Or is it wrong repository for such issue?

**Comment by QuantumGhost** [comment]
@ciembor Yes, I think you should ask the git team for implementing those hooks.
